tests/test_cli_workflows.py

```diff
@@ -1061,3 +1061,36 @@ def test_share_remove_workflow():
             )
             assert result.exit_code == 0
             assert response["message"] in result.output
+
+
+def test_share_status_workflow():
+    """Test share-status workflows."""
+    status_code = 200
+    response = {
+        "message": "is not shared with anyone",
+        "workflow_id": "string",
+        "workflow_name": "string",
+    }
```

Comment:
This is not a valid response, if the error code is 200 then there should be a `shared_with` key, and not `message`, right?